app/src/main/res/layout/fragment_match.xml

```diff
@@ -0,0 +1,153 @@
+<?xml version="1.0" encoding="utf-8"?>
+<android.support.constraint.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
not sure what this screen looks like, but looking at all the `Top_toBottomOf` constraints, a LinearLayout may be a better option?

Reply:
This is a screen for future PR, I haven't looked into this screen yet but I think we should keep this for future use (and if `LinearLayout` is better we'll use it then).